Makefile

```diff
@@ -59,6 +59,9 @@ $(BIN_DIR)/tools/protoc-gen-go: go.mod
 $(BIN_DIR)/tools/protoc-gen-go-grpc: go.mod
 	GOBIN=$(abspath $(BIN_DIR))/tools go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
 
+$(BIN_DIR)/tools/golangci-lint: golangci-lint.version
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BIN_DIR)/tools $(shell cat golangci-lint.version)
```

Comment:
This will execute whatever is hosted in that install script -- instead, use `gettool` so we only download what we need directly from the releases page.

Reply:
I switched it to use gettool. Their release for Windows only has **.zip** whereas linux/mac is **.gz**, so my current solution won't work for Windows. Should we account for Windows users as well? That would probably require a combination of the logic in `getPrometheus` and `getProtoC`.

Reply:
We should; I played with it this AM and pushed a fix for that. On a separate note, I think the `gettool` stuff should be refactored, especially with the increase in external tools.

Thanks for making the update!